services/app-api/src/postgres/contractAndRates/proto_to_db_UpdateInfoTable.ts

```diff
@@ -0,0 +1,40 @@
+import type { PrismaClient, HealthPlanRevisionTable } from '@prisma/client'
+import type { HealthPlanFormDataType } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+
+async function prepopulateUpdateInfo(
+    client: PrismaClient,
+    revision: HealthPlanRevisionTable,
+    formData: HealthPlanFormDataType
+): Promise<string | Error> {
+    try {
+        const user = await client.user.findFirst({
+            where: { email: { equals: revision.submittedBy ?? '' } },
+        })
+
+        if (!user) {
+            const error = new Error(
+                `User with email ${revision.submittedBy} does not exist. Skipping UpdateInfoTable creation.`
+            )
+            return error
+        }
+
+        console.info(`DEBUG: ${JSON.stringify(user)}`)
+        console.info(`DEBUG: ${JSON.stringify(formData.updatedAt)}`)
+        const updateInfo = await client.updateInfoTable.create({
+            data: {
+                updatedAt: formData.updatedAt,
+                updatedByID: user.id,
+                updatedReason: 'Migrated from previous system',
```

Comment:
We display the update info in the user facing UI. These update reasons have up until now always been text entered by our users. Is the plan to filter these entries out from display?

Reply:
Good call out -- I think it's something we probably want to decide on as a team. Do we want to have a default like this? Do we want to filter it out on the FE? I don't have a strong opinion here.